### PR TITLE
[doc] Add doc about how to use healthcheck endpoints

### LINKED ISSUE #10589: [Feature][Doc] Add doc about how to use healthcheck endpoints
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Add doc about how to use healthcheck endpoints

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by SbloodyS**
## Purpose of the pull request

close #10589

### CODE REVIEW

File: docs/docs/en/guide/healthcheck.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Health Check
+
+## Background
+
+Health check are designed to provide a unique way to check the health of the dolphinscheduler service. It includes the health status of modules, such as DB, cache, network, etc.
+
+## Endpoint
+
+### API-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/en/guide/healthcheck.md:11]:
```suggestion

```shell
```

File: docs/docs/en/guide/healthcheck.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:12345/dolphinscheduler/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+### Master-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/en/guide/healthcheck.md:19]:
```suggestion

```shell
```

File: docs/docs/en/guide/healthcheck.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:5679/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+### Worker-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/en/guide/healthcheck.md:27]:
```suggestion

```shell
```

File: docs/docs/en/guide/healthcheck.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:1235/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+### Alert-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/en/guide/healthcheck.md:35]:
```suggestion

```shell
```

File: docs/docs/en/guide/healthcheck.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:50053/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+
```
Comment (zhongjiajie) [inline docs/docs/en/guide/healthcheck.md]:
```suggestion

```

File: docs/docs/zh/guide/healthcheck.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# 健康检查
+
+## 背景
+
+运行状况检查旨在提供一种独特的方法来检查 Dolphinscheduler 服务的运行状况。它包括模块的运行状况，例如 DB、缓存、网络等等。
+
+## Endpoint
+
+### API-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/healthcheck.md:11]:
```suggestion

```shell
```

File: docs/docs/zh/guide/healthcheck.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:12345/dolphinscheduler/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+### Master-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/healthcheck.md:19]:
```suggestion

```shell
```

File: docs/docs/zh/guide/healthcheck.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:5679/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+### Worker-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/healthcheck.md:27]:
```suggestion

```shell
```

File: docs/docs/zh/guide/healthcheck.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:1235/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+### Alert-Server
+```shell
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/healthcheck.md:35]:
```suggestion

```shell
```

File: docs/docs/zh/guide/healthcheck.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+curl --request GET 'http://localhost:50053/actuator/health'
+
+{"status":"UP","components":{"db":{"status":"UP","details":{"database":"H2","validationQuery":"isValid()"}}}}
+```
+
+
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/healthcheck.md]:
```suggestion

```

**Comment by zhongjiajie** [comment]
We should better add a blank new line between markdown title and content

**Comment by zhongjiajie** [comment]
BTW, should we add some content to notice users have to change the host and port if their change the default value

**Comment by SbloodyS** [comment]
> We should better add a blank new line between markdown title and content

Thanks.

**Comment by SbloodyS** [comment]
> BTW, should we add some content to notice users have to change the host and port if their change the default value

I'll add it.

**Review by zhongjiajie — APPROVED** [review]
looking good and thansk

**Pull request merged**